tfhe/src/integer/gpu/client_key/radix.rs

```diff
@@ -0,0 +1,95 @@
+use crate::core_crypto::gpu::lwe_bootstrap_key::CudaLweBootstrapKey;
+use crate::core_crypto::gpu::CudaStreams;
+use crate::core_crypto::prelude::{
+    allocate_and_generate_new_lwe_packing_keyswitch_key, par_generate_lwe_bootstrap_key,
+    LweBootstrapKey,
+};
+use crate::integer::gpu::list_compression::server_keys::{
+    CudaCompressionKey, CudaDecompressionKey,
+};
+use crate::integer::gpu::server_key::CudaBootstrappingKey;
+use crate::integer::RadixClientKey;
+use crate::shortint::engine::ShortintEngine;
+use crate::shortint::list_compression::{CompressionKey, CompressionPrivateKeys};
+use crate::shortint::{ClassicPBSParameters, EncryptionKeyChoice, PBSParameters};
+
+impl RadixClientKey {
+    pub fn new_cuda_compression_decompression_keys(
```

Comment:
just as a reminder, what's the use case here ? as the BC will use the compressed keys for storage and network bandwidth usage

Reply:
I would say closureness. Assuming the user has LWE ciphertexts already in the CUDA format, they could generate compression/decompression keys direct into the GPU memory and ready to compress those ciphertexts. Without this method the user will always need to generate CPU compressed compression / decompression keys and then convert those.

Do we want to support these two possibilities?

Reply:
sure